ldap/servers/slapd/connection.c

```diff
@@ -189,6 +189,14 @@ connection_cleanup(Connection *conn)
     if (conn->c_prfd) {
         PR_Close(conn->c_prfd);
     }
+#ifdef ENABLE_EPOLL_NOTYET
```

Comment:
Just curious why this is _NOTYET? What was your concern about it?

Reply:
No concern.  I just failed to revert that when diagnosing an issue.